waftools/detections/compiler_swift.py

```diff
@@ -14,8 +14,9 @@ def __run(cmd):
         return ""
 
 def __add_swift_flags(ctx):
-    ctx.env.SWIFT_FLAGS = ('-frontend -c -sdk %s -enable-objc-interop -emit-objc-header'
-                           ' -emit-module -parse-as-library') % (ctx.env.MACOS_SDK)
+    ctx.env.SWIFT_FLAGS = ('-frontend -c -sdk %s -enable-objc-interop'
+                           ' -emit-objc-header -parse-as-library'
+                           ' -target x86_64-apple-macosx10.11') % (ctx.env.MACOS_SDK)
```

Comment:
what is this about? is this just for testing? does it break on non 10.11 OSX versions? I guess in the commit message you mean limit to **earliest** supported macOS version?

Reply:
i didn't decide yet how to handle it so this is still for testing, this is the reason why i left it as separate commit still. the vibrancy part is a feature for (actually) 10.10+ and swifts default target is 10.9. so it won't build, or i either have to fallback to the old code or have a separate code path for it.

the reason why i actually set it to 10.11 is because i at least [use](https://github.com/mpv-player/mpv/blob/master/video/out/cocoa_cb_common.swift#L148) one [10.11 feature](https://developer.apple.com/documentation/corevideo/1456927-cvdisplaylinksetoutputhandler?language=objc), which the compiler doesn't catch apparently.

i personally would like to limit the use of the new backend to 10.11+, since 10.10 was also the last OS where everything was 'alright' with the old backend. yeah it was supposed to be "earliest", seems like i didn't think much when committing.